core/src/test/java/io/questdb/test/cutlass/http/line/LineHttpSenderLoggingTest.java

```diff
@@ -52,12 +52,11 @@
 
 public class LineHttpSenderLoggingTest extends AbstractBootstrapTest {
     private static final LogCapture capture = new LogCapture();
-    private static final Class<?>[] guaranteedLoggers = new Class[]{LineHttpProcessorState.class};
 
     @Before
     @Override
     public void setUp() {
-        LogFactory.enableGuaranteedLogging(guaranteedLoggers);
+        LogFactory.enableGuaranteedLogging(TableUpdateDetails.class);
```

Comment:
why not add this class to the deleted array of the guaranteed loggers? What necessitated removal of guaranteed logging from the state class?